### ISSUE TITLE
New Version of basic-ftp throws an error.

### ISSUE THREAD

**Issue by hadi16** [patrickjuchli/basic-ftp#95]
**Describe the bug**
The new release of basic-ftp (4.0.1) has thrown an error, while the older version did not.

**Example code**
```
import {Client} from "basic-ftp";
import {config} from "dotenv";

async function uploadTest(): Promise<void> {
    const ftpClient = new Client();
    ftpClient.ftp.verbose = true;

    try {
        await ftpClient.access({
            host: process.env.FTP_HOST,
            user: process.env.FTP_USER,
            password: process.env.FTP_PASSWORD,
            secure: true,
            secureOptions: {rejectUnauthorized: false}
        });

        await ftpClient.cd('/public_html');
        await ftpClient.list();
    } catch (error) {
        console.log(error);
    }

    ftpClient.close();
}

config({path: 'package.env'});

uploadTest();
```

**Console output**

```
Connected to _____________________
< 220-#########################################################
220-Please upload your web files to the public_html directory.
220-Note that letters are case sensitive.
220-#########################################################
220 This is a private system - No anonymous login
> AUTH TLS
< 234 AUTH TLS OK.
Control socket is using: TLSv1.2
Login security: TLSv1.2
> USER _________
< 331 User ________ OK. Password required
> PASS ###
< 230-Your bandwidth usage is restricted
< 230 OK. Current restricted directory is /
> TYPE I
< 200 TYPE is now 8-bit binary
> STRU F
< 200 F OK
> OPTS UTF8 ON
< 200 OK, UTF-8 enabled
> OPTS MLST type;size;perm;modify;unix.mode;unix.owner;unix.group;unix.ownername;unix.groupname
< 200  MLST OPTS type;size;sizd;modify;UNIX.mode;UNIX.uid;UNIX.gid;unique;
> PBSZ 0
< 200 PBSZ=0
> PROT P
< 200 Data protection level set to "private"
> CWD ___________
< 250 OK. Current directory is ____________
Trying to find optimal transfer strategy...
> EPSV
< 229 Extended Passive mode OK (|||34683|)
Optimal transfer strategy found.
> MLSD
< 150 Accepted data connection
Downloading from ___________ (TLSv1.2)
< 226-Options: -a -l
< 226 34 matches total
type=cdir;sizd=4096;modify=20190922222842;UNIX.mode=0755;UNIX.uid=658;UNIX.gid=639;unique=700g4ee2d1f; .
type=pdir;sizd=4096;modify=20190626144156;UNIX.mode=0755;UNIX.uid=658;UNIX.gid=639;unique=700g4ec3423; ..
type=file;size=840;modify=20181214140342;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee2d96; 404.php
type=file;size=1229;modify=20181219144826;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee2d97; archive.php
type=dir;sizd=4096;modify=20190625171234;UNIX.mode=0755;UNIX.uid=658;UNIX.gid=639;unique=700g4f040c2; classes
type=file;size=3988;modify=20181214145540;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee2d98; comments.php
type=dir;sizd=4096;modify=20190625171234;UNIX.mode=0755;UNIX.uid=658;UNIX.gid=639;unique=700g4f226b1; fonts
type=file;size=1608;modify=20190625171237;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee2d99; footer.php
type=file;size=10035;modify=20190103053450;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee2d9a; functions.php
type=file;size=1870;modify=20190416120054;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee2d9b; header.php
type=file;size=2874;modify=20190416120054;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee2d9c; image.php
type=dir;sizd=4096;modify=20190625171234;UNIX.mode=0755;UNIX.uid=658;UNIX.gid=639;unique=700g4f44b7b; inc
type=file;size=1060;modify=20181219144826;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee2d9d; index.php
type=dir;sizd=4096;modify=20190625171234;UNIX.mode=0755;UNIX.uid=658;UNIX.gid=639;unique=700g4f634ec; js
type=file;size=151698;modify=20181214140342;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee2d9e; package-lock.json
type=file;size=1531;modify=20181214140342;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee2d9f; package.json
type=file;size=765;modify=20181214140342;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee2da0; page.php
type=file;size=219;modify=20181214140342;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee2da1; postcss.config.js
type=file;size=3949;modify=20190228141752;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee2da2; print.css
type=file;size=3320;modify=20190407164852;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee2da3; print.scss
type=file;size=1735;modify=20190508053656;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee2da4; readme.txt
type=dir;sizd=4096;modify=20190625171234;UNIX.mode=0755;UNIX.uid=658;UNIX.gid=639;unique=700g4f81ecb; sass
type=file;size=175535;modify=20181214140342;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee3409; screenshot.png
type=file;size=1344;modify=20181214140342;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee34f5; search.php
type=file;size=1785;modify=20181219145412;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee35df; single.php
type=file;size=159;modify=20190119060850;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee35e0; style-editor-customizer.css
type=file;size=158;modify=20190119060850;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee35e1; style-editor-customizer.scss
type=file;size=64981;modify=20190214061050;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee35e2; style-editor.css
type=file;size=12830;modify=20190214061050;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee35e3; style-editor.scss
type=file;size=212651;modify=20190508053656;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee35e4; style-rtl.css
type=file;size=212849;modify=20190508053656;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee35e5; style.css
type=file;size=2798;modify=20190508053656;UNIX.mode=0644;UNIX.uid=658;UNIX.gid=639;unique=700g4ee3852; style.scss
type=dir;sizd=4096;modify=20190625171234;UNIX.mode=0755;UNIX.uid=658;UNIX.gid=639;unique=700g50261bd; template-parts
type=dir;sizd=4096;modify=20190922222844;UNIX.mode=0755;UNIX.uid=658;UNIX.gid=639;unique=700g586327a; wp_custom_templates

Error: This library only supports MLSD, Unix- or DOS-style directory listing. Your FTP server seems to be using another format. You can see the transmitted listing when setting `client.ftp.verbose = true`. You can then provide a custom parser to `client.parseList`, see the documentation for details.
    at Client.parseList (________/node_modules/basic-ftp/dist/parseList.js:36:15)
    at Client._requestListWithCommand (_______________/node_modules/basic-ftp/dist/Client.js:408:21)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Which version of Node.js are you using?**
Node v10.16.3

**Additional context**
I am using a FTP server hosted by SiteGround (https://www.siteground.com/).

**Comment by patrickjuchli** [comment]
Yes, this is a bug, will look into it.

**Comment by patrickjuchli** [comment]
Fixed in 4.0.2
